pages/product-selection-opensuse-page.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+
+export class ProductSelectionPage {
+    readonly page: Page;
+    readonly productSelectionText: Locator;
+    readonly opensuseTumbleweedCheck: Locator;
+    readonly opensuseLeapCheck: Locator;
+    readonly selectButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.productSelectionText = page.getByText('Product selection');
+        this.opensuseServerType_Check = page.getByLabel(process.env.PRODUCTNAME);
+        this.selectButton = page.getByRole('button', { name: 'Select' });
+    }
+
+    async expectProductSelection() {
+        await expect(this.productSelectionText).toBeVisible(true);
+    }
+
+    async checkServerType() {
```

Comment:
using the language of the screen, `choose_opensuse_tumbleweed`

Reply:
Currently the design is to use the env variable of process.env.PRODUCTNAME to avoid hard-code product name, WDYT?

Reply:
Updated